pytripgui/canvas_vc/canvas_controller.py

```diff
@@ -80,19 +85,35 @@ def _update_canvas_view(self):
                 self._model.let.prepare_data_to_plot()
                 self._ui.plot_let(self._model.let)
 
-    def set_patient(self, patient, state):
+    def set_patient(self, patient, state: PatientGuiState):
         self._ui.clear()
-        if state is None:
-            state = ProjectionSelector()
-        self._model = PlotModel(state)
+
+        if state and state.projection_selector:
```

Comment:
explain this conditional blocks in more details, i.e. what situation projection_selector is set and in which - not

Reply:
Default value is None, so I check if it is in that condition, but I will check if it is possible for it to be None in that method

Reply:
I was more concerned why check for `projection_selector` at all in `set_patient` method.
It is not obvious from the code, hence should go into the comment

Reply:
Done